indico/modules/search/client/js/components/SearchBar.module.scss

```diff
@@ -15,3 +16,7 @@
     margin-right: 1em !important;
   }
 }
+
+.searchField > span {
```

Comment:
Just wondering, wouldn't it be better to have a global `visually-hidden` class for this, and put it directly on the span element? Before seeing this I was wondering why the span is necessary at all...

Reply:
I don't like to address styling concerns in the markup. Markup is not intended to be used for styling.

Reply:
(A.k.a. "Good ol' separation of concerns", that's how I was taught.)